libindi/drivers/telescope/lx200_OnStep.cpp

```diff
@@ -99,14 +99,18 @@ bool LX200_OnStep::initProperties()
 
     // ============== MOTION_CONTROL_TAB
 
-    IUFillNumber(&MaxSlewRateN[0], "maxSlew", "Rate", "%g", 1.0, 9.0, 1.0, 5.0);    //2.0, 9.0, 1.0, 9.0
+    IUFillNumber(&MaxSlewRateN[0], "maxSlew", "Rate", "%g", 0.0, 9.0, 1.0, 5.0);    //2.0, 9.0, 1.0, 9.0
```

Comment:
use "%.f " for format since this is integer and not double.

Reply:
Thanks Jasem,

I made the changes. Will re-issue a PR.
In the meantime I saw this magic button "squash and merge"